app/models/question.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: questions
+#
+#  id         :bigint           not null, primary key
+#  input_type :integer          default("string")
+#  text       :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+class Question < ApplicationRecord
+  enum input_type: [:string, :boolean, :integer, :array]
```

Comment:
I think `Question` should be scoped to organization? Similar to my `Form` comment, this would be using `acts_as_tenant` to scope a `Question` to an organization.

Reply:
Ok in doing this it occurred to me that it makes sense to let all of the new models in this PR be scoped under `organization` so I put a foreign key to `organization` on all of them. Does that sound reasonable to you?

Reply:
Do we need to scope them all to an org? I think it makes sense for Form and Question so we can easily access these. The rest I think we would be retrieving in the context of something else?

FormQuestion would be form.form_questions.where(form_id: x) which would be used to display an index of questions for a given form
Response would be user.submission.responses.where(form_id: x) - or I think we could say a User has responses through submissions to call user.responses.where(form_id: x), which would be used to display a user's responses to a given form

Maybe I am not thinking far enough out, but do we need to be able to retrieve all FormQuestion or all Responses or Submissions in the context of an Org? If so, then I think the belongs_to makes sense? Otherwise I don't think we need to scope them all. What do you think?

Reply:
I can go either way on this. In the past when I've done multi-tenant apps (which this _kinda_ is but kinda isn't) we were always very careful to scope everything to the tenant so that we don't accidentally leak data between tenants. But I think the stakes are lower here and the possibility of leakage seems lower too.

I suppose if we ever began a query with Response or FormQuestion then it'd pose a risk but that seems like enough of an edge case I'm happy to keep those unscoped. Should I go ahead and unscope those models off organization?

Reply:
Yes I do see your point, though. The scale we are at anticipating (not huge) and the nature of the development process (open source) I think that the chance of data leak is greater than the negative performance effects. I.e., at some stage a query with Response or FormQuestions might be allowed into production, causing a bit of mayhem. So, I agree with your approach to scope it. Thanks for the clarity on that!